src/input_package/README.md

```diff
@@ -0,0 +1,57 @@
+# Input Package User Guide
+This package provides a wrapper around the basic pygame input functionality to enable easy to use input system for games.
+Main features
+- Easy to use
+- Multiple keys for same button action (ex. ctrl or F to shoot)
```

Comment:
Multiple keys for same button action (ex. ctrl or F to shoot)?? (I'm assuming this is from somwhere else :p )